### ISSUE TITLE
[Feature Request]: Integrate Laravel into the new project wizard

### ISSUE THREAD

**Issue by abenerd** [laravel-idea/plugin#585]
### Feature Description

Add option to create a new Laravel project via the wizard with all the options and prompts of the Laravel installer.

**Comment by adelf** [comment]
https://twitter.com/laravel_idea/status/1469163927435108353

I already have this git branch... I hope it will be implemented this summer.

**Comment by gorbunov** [comment]
welp, it wasn't that branch after all... )

**Comment by adelf** [comment]
Yep. It's finally implemented by @gorbunov in the 10.2 update. Thanks!)
